.github/workflows/add-issues-and-prs-to-project.yml

```diff
@@ -0,0 +1,19 @@
+name: Add issues and pull requests to project
```

Comment:
Looks weird right now
![image](https://user-images.githubusercontent.com/9084735/162461523-9f924ba9-bf93-46e1-804f-09eba02a49a5.png)


Maybe make this name shorter and add a name to the job.

Other ones look cleaner for example
![image](https://user-images.githubusercontent.com/9084735/162461814-f4510b79-997e-4287-9fae-ab92d41ced64.png)

Reply:
added